### PR TITLE
Add simple healthcheck to app container nginx template

### PR DESCRIPTION
**Pull request by andrewthauer**
## Hey, I just made a Pull Request!

The default `app` docker container setup does not have an explicit healthcheck. This adds a basic `/healthcheck` route in the nginx config template to support a simple health/liveness check.

#### :heavy_check_mark: Checklist

- [x] All tests are passing `yarn test`
- [x] Screenshots attached (for UI changes)
- [x] Relevant documentation updated
- [x] Prettier run on changed files
- [x] Tests added for new functionality
- [x] Regression tests added for bug fixes

### CODE REVIEW

File: docker/default.conf.template [hunk 1]
```diff
@@ -11,6 +11,11 @@ server {
         try_files $uri /index.html;
     }
 
+    location /healthcheck {
+        types { } default_type "text/plain; charset=utf-8";
+        return 200 "ok\n";
```
Comment (Rugvip) [inline docker/default.conf.template:16]: Think we could go with a simple `return 204;` to keep things slim?

**Review by Rugvip — COMMENTED** [review]
Yep, let's have this, thanks! :grin:

**Comment by andrewthauer** [comment]
@Rugvip - Would you mind taking another look at this?

File: docker/default.conf.template [hunk 2]
```diff
@@ -11,6 +11,11 @@ server {
         try_files $uri /index.html;
     }
 
+    location /healthcheck {
+        types { } default_type "text/plain; charset=utf-8";
```
Comment (Rugvip) [inline docker/default.conf.template:15]: No need for this line now

**Review by Rugvip — COMMENTED** [review]
Ah shite, missed this one 😅

File: docker/default.conf.template (hunk 2 above)
Comment (andrewthauer) [reply docker/default.conf.template:15]: Good point

**Review by Rugvip — APPROVED** [review]
:shipit:

**Pull request merged**